adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -810,6 +811,8 @@ private IWindowComponent wrapActivity(final Activity activity) {
         }
 
         return new IWindowComponent() {
+            //CHECKSTYLE:OFF: checkstyle:MemberName
+            //CHECKSTYLE:OFF: checkstyle:VisibilityModifier
             Activity mRefActivity = activity;
```

Comment:
Yes, I understood. I addressed @addev-ashish 's commit and run the test. My previous reply is trying to answer @addev-ashish 's question on "if doing this would break anything". :)